lib/msf/base/config.rb

```diff
@@ -300,6 +307,14 @@ def history_file
     config_directory + FileSep + "history"
   end
 
+  def met_history
+    config_directory + FileSep + "met_history"
```

Comment:
I'd really prefer to see this not abbreviated. It's not something the user is going to have to type so I think clarity would be preferred over brevity. For example, not being familiar with what's going on here could lead someone to wonder is this the history file for **Met**asploit, **Met**erpreter, **Met**asm, or **Met**tle? We have a lot of Met- things.

Reply:
Okay I'll change this and give a more sensible name :+1: